Fix #949 - Truncate email address to prevent overlap

codemist opened the pull request:
Preview:
![image](https://user-images.githubusercontent.com/13066134/126845673-a5003a59-e5b7-41d2-9b8a-5dba95f3a2bf.png)

maxxcrawford commented on static/scss/partials/dashboard.scss:
```diff
@@ -5,6 +5,8 @@
 
 .mpp-dashbaord-header-name {
     font-weight: 700;
+    width:100%;
```
Is the `width` here necessary?

codemist replied:
Yes, it helps keep the word-wrap property valid on mobile.
https://user-images.githubusercontent.com/13066134/127021103-30bc7aff-5b4b-4c35-9917-b5d950b56b26.mov

codemist commented:
I changed the container to `align-items: center;` so that when the email breaks off into two lines the dashboard stats remains centered. I also added a `padding-right` property to increase the gutter space between the header title and the stats (visually nicer in my opinion).

Preview:
![image](https://user-images.githubusercontent.com/13066134/127021660-6e74ddc2-9fe5-4e24-a2df-09f5bf68d8de.png)

maxxcrawford commented on static/scss/partials/dashboard.scss:
```diff
@@ -5,6 +5,10 @@
     font-weight: 700;
+    width: 100%;
+    word-wrap: break-word;
+    padding-right:5%;
```
nit: whitespace

And also – percentage padding feels funky here. Does it work with a token? 

```suggestion
    padding-right: 5%;
```

codemist replied:
Switched out the percentage for `$spacing-sm` instead! 😄

maxxcrawford commented on static/scss/partials/dashboard.scss:
```diff
@@ -138,7 +142,7 @@
     .mpp-dashbaord-header-container {
         display: flex;
-        align-items: flex-end;
+        align-items:center;
```
With centering, does it look the same when only on one line?  Same question for three/four lines? I want to understand how the different length levels impact this layout. 

nit: whitespace

```suggestion
        align-items: center;
```

codemist commented:
Without `align-items: center;` :
![image](https://user-images.githubusercontent.com/13066134/127231839-4d85dca2-82e3-4062-be20-d2a998fe82d2.png)


With it:

One line:
<img width="912" alt="image" src="https://user-images.githubusercontent.com/13066134/127231548-f8c62adf-dc52-4929-a4dd-c26922d93d8f.png">

Two lines:
<img width="1001" alt="image" src="https://user-images.githubusercontent.com/13066134/127231568-1286808f-cf98-47b9-8cbc-d51d58b944f0.png">

Three lines:
<img width="951" alt="image" src="https://user-images.githubusercontent.com/13066134/127231600-819e454e-61b3-4aed-8eb6-8d300a64d2c4.png">

Four lines:
<img width="904" alt="image" src="https://user-images.githubusercontent.com/13066134/127231647-354ae801-21a0-4fdc-a342-edbbdecdf6e3.png">

maxxcrawford reviewed (approved):
Nice work!
